core/src/main/java/org/dcsa/conformance/core/party/CustomJsonPointer.java

```diff
@@ -13,7 +13,8 @@ private CustomJsonPointer() {
     // Private constructor to prevent instantiation
   }
 
-  public static List<JsonNode> findMatchingNodes(JsonNode rootNode, String pathExpression, BiPredicate<JsonNode,String> condition, String paramValue) {
+  public static List<JsonNode> findMatchingNodes(JsonNode rootNode, String pathExpression,
+                                                 BiPredicate<JsonNode,String> condition, String paramValue) {
```

Comment:
Which formatting are you using, because when I format it, it formats differently. These differences should be avoided.

Reply:
[*]
charset = utf-8
end_of_line = lf
indent_style = space
indent_size = 2
trim_trailing_whitespace = true
insert_final_newline = true